tg.js

```diff
@@ -240,29 +240,29 @@ settings.crosshair.gap Gap
 settings.crosshair.dot Tuldok
 settings.crosshair.opacity.use Gumamit ng Pasadyang Opacity
 settings.crosshair.opacity.custom Pasadyang Opacity
-settings.crosshairm.header Crosshair (Throwing)
-settings.crosshairm.type Type
-settings.crosshairm.type.1 Shapes
-settings.crosshairm.type.2 Image
-settings.crosshairm.type.3 Precision
-settings.crosshairm.style Style
```

Comment:
Can we just make this "Hitsura" or "Hugis" instead of "Istilo"

Reply:
That would interfere and cause confusion with 245 which is already "Shapes" directly "Hugis".

Reply:
Oh ok, how about "hitsura"?

Reply:
Im not entirely sure about it, it may alternately mean "Form" if you used "Itsure" but I can consider it, lemme ask other reviewers if they got the time.

Reply:
@callmedandan, Itsura seems to lean more towards "Appearance" than "Form" or "Style" so I think we can go with a shortened "Istilo" called "Stilo" if that's fine with you.